en/flight_modes/position_mc.md

```diff
@@ -1,17 +1,14 @@
 # Position Flight Mode (Multicopter)
 
-![GPS](../../images/flight_modes/GPS_s.png)
+[<img src="../../assets/site/difficulty_1.svg" title="Difficulty (Easiest)" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
 
-*Position* mode is an assisted RC mode, in which the vehicle can easily be brought to a stop with a level attitude (irrespective of wind and other forces), and in which the flight controls are intuitive and straightforward. 
+*Position* mode is an easy-to-fly RC mode in which the vehicle can be "locked" to a position in 3D space (irrespective of wind and other forces) by releasing (centering) the RC sticks.
 
-> **Tip** Position mode is the safest manual mode for new fliers.
+> **Tip** Position mode is the safest manual mode for new fliers. It is much like [Altitude](../flight_modes/altitude.md) mode, but additionally stabilizes horizontal position.

```

Comment:
Hmm, I would not say that. I would rather say that Altitude is much like Stabilized/Manual, except that the altitude is stabilized as well (the roll/stick input is treated the same way in both modes).
Whereas in Position mode the velocity is controlled, which from a flight experience is quite a bit different from the direct tilt angle in Altitude (for example letting go of the stick in position mode with a high velocity, the vehicle will actively brake. In Altitude mode, it continues and just slows down through air resistance).